### ISSUE TITLE
Latest git code compilation failure.

### ISSUE THREAD

**Issue by Alion548** [shadowsocks/shadowsocks-rust#749]
```shell
error[E0635]: unknown feature `llvm_asm`
 --> /root/.cargo/registry/src/github.com-1ecc6299db9ec823/crypto2-0.1.2/src/lib.rs:2:21
  |
2 | #![feature(stdsimd, llvm_asm)]
  |                     ^^^^^^^^

For more information about this error, try `rustc --explain E0635`.
error: could not compile `crypto2` due to previous error
```
```shell
export RUSTFLAGS="-C target-cpu=native" && cargo +nightly build --release --features "stream-cipher" --features "local-redir" --features "local-dns" --features "jemalloc"

nightly-x86_64-unknown-linux-gnu unchanged - rustc 1.60.0-nightly (5e57faa78 2022-01-19)
```

**Comment by dev4u** [comment]
需要回退rust的版本

**Comment by zonyitoo** [comment]
Yes, use the Rust version recommended in [rust-toolchain](https://github.com/shadowsocks/shadowsocks-rust/blob/master/rust-toolchain). It will be fixed later when crypto2 releases 2.0.

**Comment by paseaf** [comment]
just a note:
`nightly-2022-01-19` didn't work for me.
`nightly-2021-12-14` worked.
